app/assets/javascripts/services/ChatStates.js

```diff
@@ -219,6 +219,14 @@ export class EngagedState {
         }
     }
 
+    _suppressMessage(msg, transcript) {
```

Comment:
I think this should be called chatRoomMemberLost as it doesn't always suppress a message